src/main/scala/Chisel/Tester.scala

```diff
@@ -1,39 +0,0 @@
-// See LICENSE for license details.
```

Comment:
We are releasing chisel-testers and I will add a LICENCE file to that repo. Not sure what needs to be done in the README for this.